app/views/eventRegistrations.html

```diff
@@ -44,17 +44,34 @@ <h2 class="page-title" translate>Registrations</h2>
           </button>
           <button class="btn btn-primary" ng-click="refreshRegistrations()"><i class="fa fa-refresh"></i> <translate>Refresh</translate></button>
           <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
-            <translate>Columns</translate>
+            <span class="hidden-xs"><translate>Display</translate></span><translate> Columns</translate>
```

Comment:
You could probably also split it onto a new line there to make it easier to read since the line got quite a bit longer.